iommi/table__tests.py

```diff
@@ -954,6 +954,20 @@ def test_bulk_delete_all_uses_original_rows():
     assert list(TFoo.objects.all().order_by('a').values_list('a', flat=True)) == [2]
 
 
+@pytest.mark.django_db
+def test_bulk_include_false():
+
+    table = Table(
+        auto__rows=TFoo.objects.all(),
+        columns__a__bulk__include=True,
+        bulk__include=False,
+    ).bind(request=req('get'))
+
+    assert table.bulk == None
+    table.__html__()
+    assert True
```

Comment:
assert True? Is this just to show that there's no crash? Then I'd put a comment before `table.__html__()` saying that's why we're rendering and remove the assert.